model/db.ts

```diff
@@ -1,86 +1,194 @@
-export interface Lote {
+import {nanoid} from "nanoid"
+import { Kv } from "../test/db_test.ts";
+
+export interface Lote 
+{
     id: string;
     peso: number;
     volume: number;
 }
 
-export interface Vehiculo {
+export interface Vehiculo 
+{
     id: string;
     pesoMax: number;
     volumeMax: number;
-    lotes: string[];
 }
 
-export interface Pedido {
+export interface Pedido 
+{
```

Comment:
¿Por qué has cambiado esta convención?
Conviene en todo caso que estas cosas las hagáis antes del PR, porque genera cambios en ficheros que hay que evaluar de todas formas.

Reply:
El cambio no ha sido por alguna causa específica sino más bien porque se usaban las dos convenciones y se ha querido unificar. Perfecto, para las próximas veces lo tendré en cuenta.